test/remote/session_tests.js

```diff
@@ -241,7 +400,7 @@ describe('remote session', function() {
         var email = server.uniqueEmail()
         var password = 'testx'
         var uid = null
-        return Client.create(config.publicUrl, email, password)
+        return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
```

Comment:
This was depending on `skipForNewAccounts` being `true`, and has been tweaked to work with it `false`.